bump v8.1.0 as npm minimum version

ref: https://github.com/aws-samples/baseline-environment-on-aws/pull/3
I confirmed it is required at least `v8.1.0` npm version to run correctly. We need at least npm/bin-links v2.3.0. I tested:

* `npm@7.24.2` -> Fail
* `npm@8.0.0` -> Fail
* `npm@8.1.0` -> OK
* `npm@8.1.3` -> OK

Error log:

```console
$ sudo yum insatll -y git
$ curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.37.2/install.sh | bash
$ . ~/.nvm/nvm.sh
$ nvm install --lts=fermium
$ nvm use --lts=fermium
$ node -e "console.log('Running Node.js ' + process.version)"
$ npm i -g npm@7.15.1
$ cd
$ git clone https://github.com/aws-samples/baseline-environment-on-aws.git
$ cd baseline-environment-on-aws
$ npm ci
npm WARN deprecated uuid@3.3.2: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm notice
npm notice New major version of npm available! 7.15.1 -> 8.1.3
npm notice Changelog: https://github.com/npm/cli/releases/tag/v8.1.3
npm notice Run npm install -g npm@8.1.3 to update!
npm notice
npm ERR! code ENOENT
npm ERR! syscall chmod
npm ERR! path /home/ssm-user/baseline-environment-on-aws/node_modules/blea-base-ct-audit/bin/blea-base-ct-audit.js
npm ERR! errno -2
npm ERR! enoent ENOENT: no such file or directory, chmod '/home/ssm-user/baseline-environment-on-aws/node_modules/blea-base-ct-audit/bin/blea-base-ct-audit.js'
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/ssm-user/.npm/_logs/2021-11-10T13_08_51_654Z-debug.log
```

--- a/README.md
+++ b/README.md
@@ -76,10 +76,10 @@ Describe the steps to deploy. When deploying only, it is not necessary to build 
 Use the following runtimes: Follow the instructions for each OS to install.
 
 - [Node.js](https://nodejs.org/) (>= `14.0.0`)
-  - `npm` (>= `7.0.0`)
+  - `npm` (>= `8.1.0`)
 - [Git](https://git-scm.com/)
 
-npm requires 7.0.0 or higher because it uses workspaces. Please install the latest version as follows.
+npm requires 8.1.0 or higher because it uses workspaces. Please install the latest version as follows.
 
 ```sh
 npm install -g npm

--- a/README_ja.md
+++ b/README_ja.md
@@ -76,10 +76,10 @@ Jump to | [Changelog](CHANGELOG.md) | [HowTo](doc/HowTo_ja.md) | [マルチア�
 以下のランタイムを使用します。各 OS ごとの手順に従いインストールしてください。
 
 - [Node.js](https://nodejs.org/) (>= `14.0.0`)
-  - `npm` (>= `7.0.0`)
+  - `npm` (>= `8.1.0`)
 - [Git](https://git-scm.com/)
 
-npm は workspaces を使用するため 7.0.0 以上が必要です。最新バージョンは以下のようにしてインストールしてください。
+npm は workspaces を使用するため 8.1.0 以上が必要です。最新バージョンは以下のようにしてインストールしてください。
 
 ```sh
 npm install -g npm

--- a/buildspec.yaml
+++ b/buildspec.yaml
@@ -3,7 +3,7 @@ version: 0.2
 phases:
   install:
     commands:
-      # requires npm >7
+      # requires npm >=8.1.0
       - npm i -g npm
   pre_build:
     commands:

--- a/package.json
+++ b/package.json
@@ -10,7 +10,7 @@
   "license": "MIT-0",
   "engines": {
     "node": ">=14",
-    "npm": ">=7"
+    "npm": ">=8.1.0"
   },
   "workspaces": [
     "usecases/*",

--- a/tools/cicd/README.md
+++ b/tools/cicd/README.md
@@ -90,7 +90,7 @@ version: 0.2
 phases:
   install:
     commands:
-      # requires npm >7
+      # requires npm >=8.1.0
       - npm i -g npm
   pre_build:
     commands:

--- a/tools/cicd/README_ja.md
+++ b/tools/cicd/README_ja.md
@@ -91,7 +91,7 @@ version: 0.2
 phases:
   install:
     commands:
-      # requires npm >7
+      # requires npm >=8.1.0
       - npm i -g npm
   pre_build:
     commands:

